lib/OpenQA/CLI/api.pm

```diff
@@ -114,18 +128,20 @@ sub command ($self, @args) {
       schema=JobTemplates-01.yaml preview=0 --param-file template=foo.yaml
 
     # Post job template (from JSON file)
-    openqa-cli api --data-file form.json -X POST job_templates_scheduling/1
+    openqa-cli api -f --data-file form.json -X POST job_templates_scheduling/1
 
   Options:
         --apibase <path>          API base, defaults to /api/v1
         --apikey <key>            API key
         --apisecret <secret>      API secret
     -a, --header <name:value>     One or more additional HTTP headers
-    -D, --data-file <path>        Load content to send with request from file
+    -D, --data-file <path>        Load content to send with request from file.
+                                  Works with when -f is provided
     -d, --data <string>           Content to send with request, alternatively
                                   you can also pipe data to openqa-cli
     -f, --form                    Turn JSON object into form parameters
         --host <host>             Target host, defaults to http://localhost
+    -H, --hook-scripts            Use a hook script!! TDB
```

Comment:
I guess you mean TBD but more importantly the parameter name should most likely be:

```suggestion
    -H, --hook-script             Use a hook script!! TBD
```

Reply:
This parameter is not needed

```
      'pre-script-hook=s' => \my $prehooksrc,
      'post-script-hook=s' => \my $posthooksrc,
```

during the session we also spoke about not having short options for these.